public/stylesheets/styles.css

```diff
@@ -82,6 +82,7 @@ footer{
   text-align: center;
 }
 footer p {
+  font-family: 'Crimson Pro', serif;
```

Comment:
this is just my personal opinion, but would prefer having the old font because the footer is seen on every page but only the index page uses this font :D
(or you can add this css rule to indexPage.css and put the selector `#landing-page footer p` so it will only be seen on the index page)

Reply:
thanks i will do only for landing page, totally forgot about other pages lol